R/timeseriessummary-data.R

```diff
@@ -103,8 +105,10 @@ constructTSDetails <- function(reportData, timezone){
         changeNote <- TRUE
         itValue <- paste(itValue, "*")
       }
-      tsAttrs <- rbind(tsAttrs, data.frame(label="Interpolation", value=itValue, indent=8, stringsAsFactors = FALSE))
     }
+    tsAttrs <- rbind(tsAttrs, data.frame(label="Interpolation", value=itValue, indent=8, stringsAsFactors = FALSE))
+    
+    #Continued Time Series Attributes
```

Comment:
just being picky about language - this makes it sounds like it might be referencing "continuous time series" but I think you're just commenting that the following code is just continuing with time series attributes?